portfolio/src/main/webapp/owl.carousel/.gitattributes

```diff
@@ -0,0 +1,5 @@
+* text eol=lf
```

Comment:
This file should be in the "add Owl" PR, not this one.

Reply:
it is also in the add Owl PR, I believe